app/services/idv/actions/ipp/redo_state_id_action.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module Actions
+    module Ipp
+      class RedoStateIdAction < Idv::Steps::DocAuthBaseStep
+        def call
+          mark_step_incomplete(:state_id)
```

Comment:
Do we have a plan for further changes or wiring for the IPP `RedoStateIdAction` step?